internal/agent/hooks/kcrypt_uki.go

```diff
@@ -143,5 +149,32 @@ func (k KcryptUKI) Run(c config.Config, _ v1.Spec) error {
 	if err != nil {
 		return err
 	}
+
+	// Copy logs to persistent partition
+	c.Logger.Debug("Copying logs to persistent partition")
+	err = machine.Mount(constants.PersistentLabel, constants.PersistentDir)
+	if err != nil {
+		c.Logger.Errorf("could not mount persistent partition: %s", err)
+		return nil
+	}
+
+	// Create the directory on persistent if it doesn't exist
+	exists, _ := fsutils.Exists(c.Fs, filepath.Join(constants.PersistentDir, ".state", "var-log.bind"))
```

Comment:
I think we don't need ot check if it exists

https://github.com/kairos-io/kairos-agent/blob/d139550b545d8d200094b2c57cf57a3d050fbe32/pkg/utils/fs/fs.go#L75

Reply:
good one